src/react-components/change-scene-dialog.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styles from "../assets/stylesheets/change-scene-dialog.scss";
+import cx from "classnames";
+import DialogContainer from "./dialog-container.js";
+import { handleTextFieldFocus, handleTextFieldBlur } from "../utils/focus-utils";
+import { WithHoverSound } from "./wrap-with-audio";
+
+export default class ChangeSceneDialog extends Component {
+  state = {
+    url: ""
+  };
+
+  static propTypes = {
+    onChange: PropTypes.func,
+    onClose: PropTypes.func
+  };
+
+  onUrlChange = e => {
+    this.setState({
+      url: e.target && e.target.value
+    });
+  };
+
+  reset = e => {
+    e.preventDefault();
+    this.setState({ url: "" });
+  };
+
+  onChangeClicked = e => {
+    e.preventDefault();
+
+    if (this.state.url) {
+      this.props.onChange(this.state.url);
+    }
+
+    this.props.onClose();
+  };
+
+  render() {
+    const { onChange, onClose, ...other } = this.props; // eslint-disable-line no-unused-vars
+
+    return (
+      <DialogContainer title="Custom Scene" onClose={this.props.onClose} {...other}>
+        <div>
+          <div>
+            <p>
+              Paste a URL to a{" "}
+              <a href="/spoke" target="_blank" rel="noopener noreferrer">
+                Spoke
+              </a>{" "}
+              scene or a URL to a{" "}
+              <a href="https://en.wikipedia.org/wiki/GlTF#GLB" target="_blank" rel="noopener noreferrer">
+                GLB
+              </a>.
+            </p>
+            <p>
+              Want to create your own scene? Check out{" "}
+              <a href="/spoke" target="_blank" rel="noopener noreferrer">
+                Spoke
+              </a>!
+            </p>
+          </div>
+          <form onSubmit={this.onChangeClicked}>
+            <div className={styles.changeSceneForm}>
+              <div className={styles.inputBorder}>
+                <input
+                  className={cx(styles.leftSideOfInput)}
+                  placeholder="Scene URL"
+                  onFocus={e => handleTextFieldFocus(e.target)}
+                  onBlur={() => handleTextFieldBlur()}
+                  onChange={this.onUrlChange}
+                  type="url"
+                  value={this.state.url}
+                />
+              </div>
+              <div className={styles.buttons}>
+                <WithHoverSound>
```

Comment:
I think the plan was to remove these `WithHoverSound` wrappers entirely at some point, since we're not actually playing audio on 2D UI anymore, so probably best not to add more of them.

Reply:
i've been just 'going with the flow' assuming there will be some monster PR ripping these out eventually :) i will remove the ones I added though.